recipes/windows-terminal/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "windows-terminal" %}
+{% set version = "1.17.11461" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/microsoft/terminal/releases/download/v{{ version }}.0/Microsoft.WindowsTerminal_{{ version }}.0_x64.zip
+    sha256: f2b1539649d17752888d7944f97d6372f8d48eb1ceb024501df8d8e9d3352f25
```

Comment:
Is this binary redistributable?

Reply:
I would say yes since the windows terminal license is MIT https://github.com/microsoft/terminal/blob/main/LICENSE

Reply:
That's for the source code. Binary releases can have extra conditions.

Reply:
Do you mean because of bundled libraries inside of the release?

Reply:
No. Binary of an MIT licensed source code can be proprietary.

Reply:
https://github.com/microsoft/terminal/issues/15740#issuecomment-1648592594 Upstream clarified that redistributing Windows Terminal is allowed under their license

Reply:
@isuruf The issue thread reads like an "OK" from upstream. Would thus suffice for us or should we request some statement in the repo?

Reply:
It's good enough for me. Maybe add a link to the discussion in meta.yaml?